lib/Source.php

```diff
@@ -116,13 +116,26 @@ public function delete($params = null, $options = null)
      * @param array|null $params
      * @param array|string|null $options
      *
-     * @return BankAccount The verified bank account.
+     * @return Collection The list of source transactions.
+     */
+    public function sourceTransactions($params = null, $options = null)
+    {
+        $url = $this->instanceUrl() . '/source_transactions';
+        list($response, $opts) = $this->_request('get', $url, $params, $options);
+        $this->refreshFrom($response, $opts);
+        return $this;
```

Comment:
Is this supposed to mutate the source object?

Reply:
Oops! Absolutely not -- this endpoint returns a collection of `source_transaction` objects, not the `source` object itself. This is now fixed.